# HG changeset patch
# User Sean Farley <sean@mcs.anl.gov>
# Date 1332269669 18000
#      Tue Mar 20 13:54:29 2012 -0500
# Node ID bc20de47ec7e09494907774a7988b5948ce8e22a
# Parent  a85282f9ca5f07f4d20bdaf4f9f79726fd1235ee
cmake: remove developer's home directory from source

diff -r a85282f9ca5f -r bc20de47ec7e programs/CMakeLists.txt
--- a/programs/CMakeLists.txt	Mon Apr 08 16:49:34 2013 +0000
+++ b/programs/CMakeLists.txt	Tue Mar 20 13:54:29 2012 -0500
@@ -1,8 +1,7 @@
 # These programs use internal metis data structures.
 include_directories(../libmetis)
-link_directories(/home/karypis/local/lib)
 # Build program.
 add_executable(gpmetis gpmetis.c cmdline_gpmetis.c io.c stat.c)
 add_executable(ndmetis ndmetis.c cmdline_ndmetis.c io.c smbfactor.c)
 add_executable(mpmetis mpmetis.c cmdline_mpmetis.c io.c stat.c)
 add_executable(m2gmetis m2gmetis.c cmdline_m2gmetis.c io.c)
